site/in-dev/export_import.md

```diff
@@ -15,26 +15,26 @@ such as:
 ## Usage
 
 The Nessie Quarkus CLI too requires direct access to the database used by Nessie. The executable is
-named `nessie-quarkus-cli-x.y.z-runner.jar` and can be downloaded from the
+named `nessie-admin-cli-x.y.z-runner.jar` and can be downloaded from the
 [release page on GitHub](https://github.com/projectnessie/nessie/releases) and is available
-for Nessie 0.43.0 or newer.
+for Nessie 0.83.0 or newer.
```

Comment:
Similar, a note for 0.43..0.82

Reply:
Actually, no need to mention a verison here - this page is per release-version

Reply:
updated